gematria/datasets/python/exegesis_benchmark.cc

```diff
@@ -67,22 +68,73 @@ PYBIND11_MODULE(exegesis_benchmark, m) {
   py::class_<BenchmarkCode>(m, "BenchmarkCode");
 
   py::class_<ExegesisBenchmark>(m, "ExegesisBenchmark")
-      .def("create",
-           []() -> absl::StatusOr<std::unique_ptr<ExegesisBenchmark>> {
-             InitializeForExegesisOnce();
-
-             return LlvmExpectedToStatusOr(ExegesisBenchmark::create());
-           })
-      .def("process_annotated_block",
-           [](ExegesisBenchmark& Self,
-              const BlockWithExecutionAnnotations& Annotations) {
-             return LlvmExpectedToStatusOr(Self.processAnnotatedBlock(
-                 Annotations.block_hex(), Annotations.execution_annotations()));
-           })
-      .def("benchmark_basic_block", [](ExegesisBenchmark& Self,
-                                       const BenchmarkCode& InputBenchmark) {
-        return LlvmExpectedToStatusOr(Self.benchmarkBasicBlock(InputBenchmark));
-      });
+      .def(
+          "create",
+          []() -> absl::StatusOr<std::unique_ptr<ExegesisBenchmark>> {
+            InitializeForExegesisOnce();
+
+            return LlvmExpectedToStatusOr(ExegesisBenchmark::create());
+          },
+          R"(Creates a ExegesisBenchmark Instance.
+
+          Does the necessary initialization to run Exegesis and then creates
```

Comment:
What happens if we call this twice -- does it do this initialisation again, or only the first time it's called?

Reply:
Clarified.